README.md

```diff
@@ -15,6 +15,7 @@ The code here should work on Julia 0.6. It has extensive unit tests, but it has
 The `@save` and `@load` macros are the simplest way to interact with a JLD2 file. The `@save` macro writes one or more variables from the current scope to the JLD file. For example:
 
 ```julia
+using JLD2, FileIO
```

Comment:
True. I was just using it as another reminder, but if you prefer I'll get rid of it.